client/my-sites/plugins/plugin-sections/index.jsx

```diff
@@ -107,9 +114,59 @@ class PluginSections extends React.Component {
 					textOnly: true,
 				} ),
 			},
+			{
+				key: 'faq',
+				title: this.props.translate( 'FAQs', {
+					context: 'Navigation item',
+					textOnly: true,
+				} ),
+			},
 		];
 	};
 
+	getWpcomSupportContent = () => {
+		const supportedAuthors = [ 'Automattic', 'WooCommerce' ];
+		const { translate, plugin } = this.props;
+
+		if ( supportedAuthors.indexOf( plugin.author_name ) > -1 ) {
+			return;
+		}
+
+		const linkToAuthor = (
+			<ExternalLink href={ safeProtocolUrl( plugin.author_url ) } target="_blank">
+				{ plugin.author_url }
+			</ExternalLink>
+		);
+
+		const linkToSupportForum = (
+			<ExternalLink
+				href={ safeProtocolUrl( 'https://wordpress.org/support/plugin/' + plugin.slug ) }
+				target="_blank"
+			>
+				{ 'https://wordpress.org/support/plugin/' + plugin.slug }
+			</ExternalLink>
+		);
+
+		return (
+			<div className="plugin-sections__support">
+				<h3>{ translate( 'Support' ) }</h3>
+				<p>
+					{ translate(
+						'Support for this plugin is provided by the plugin author. You may find additional documentation at the following websites:'
+					) }
+				</p>
+				<ul>
+					<li>
+						<span>{ translate( 'Author website:' ) }</span> { linkToAuthor }
+					</li>
+					<li>
+						<span>{ translate( 'Support forum:' ) }</span> { linkToSupportForum }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **74** times:
`translate( 'Support forum' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/support-section-to-plugins)